src/core/demangle.d

```diff
@@ -2563,6 +2573,19 @@ version(unittest)
         static assert( r == table[i][1],
                 "demangled \"" ~ table[i][0] ~ "\" as \"" ~ r ~ "\" but expected \"" ~ table[i][1] ~ "\"");
     }
+
+    {
+        auto dst = new char[50];
+        auto example = table[$-2];
```

Comment:
`$-2` is not so good to use here, it will silently test something else if other symbols are added to the table.

Reply:
updated